### DISCUSSION TITLE
Highlight a residue in a protein structure

### DISCUSSION THREAD

**Discussion by conradshyu** [molstar/molstar#254] [Q&A]
Hi,
Is there a way to change the representation for a residue in a protein structure? I am trying to show the mutation (amino acid) on a protein structure. Thank you in advance for your help.

**Comment by dsehnal** [comment]
Hi,

yeah, it's possible.

Just turn on the selection mode and add a new component:

![image](https://user-images.githubusercontent.com/21145776/129546875-cd3f4e13-f3aa-483f-bb30-216ec3451580.png)

It's also possible to subtract the residue from the cartoon representation:

![image](https://user-images.githubusercontent.com/21145776/129546992-932d8097-6386-4cef-9377-3a029506cf28.png)

------------

If you want this in code, for example https://github.com/molstar/molstar/issues/250 could be a good start: you need to create a of the residue you want to highlight and then create your representation based on that.
